Automattic/woocommerce-payments
# Rename and reorder E2E tests into two groups of merchant and shopper

#### Changes proposed in this Pull Request





In order to have a clean display of all tests separated into merchant and shopper flows, I have renamed and reordered all the tests. Now we will have clean display of all of them and we could quickly find the specific test by looking at the parent folder and the name structure.





#### Testing instructions



- Make sure the GitHub Actions checks have passed, optionally run the e2e test suite locally.
